tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -296,13 +297,13 @@ def kms_decrypt(config, logger, session, encrypted_field):
                     'Plaintext']
         except (TypeError, base64.binascii.Error) as e:
             logger.warning(
-                "Error: %s Unable to base64 decode %s, will assume plaintext.",
+                "Error: %s Unable to base64 decode %s, will assume plaintext." %
```

Comment:
py std logging will lazy  render these if passed in as method arguments based on configured logging level.